index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Library App</title>
```

Comment:
Can you please give it a more descriptive title so the SEO Police are off our backs :D

Reply:
7f5cab0454e7f5dba6d6e5c12206c51994b480cf